### ISSUE TITLE
Parental controls don't copy over from the user template

### ISSUE THREAD

**Issue by alexbgameson** [hrfee/jfa-go#349]
#### Read the [FAQ](https://wiki.jfa-go.com/docs/faq/) first!

**Describe the bug**

Parental controls don't copy over from the user template.

**Platform/Version**

Version 0.5.1

**Comment by hrfee** [comment]
Hi,
What settings specifically don't copy over? Having a look on the latest Jellyfin I can see a few missing fields, namely the "Allow items with tags" section. I'll update the API client with all the new sections now, after which you'll just need to re-source your profile/user template.

**Comment by DesertCookie** [comment]
I've just encountered the same issue and don't think it's fixed yet.

I created a template account in Jelyfin I called `jfa_TAGNAME`. It only allows the user to see media with `TAGNAME` set in "Allow items with tags".  
Next, I made a new user profile in JFA-GO and chose this account as source.  
Finally, creating an invite link using this template and creating a new account, this account can see ALL media. Looking at the account in the admin control panel, it has no entries in "Allow items with tags".
